src/applications/appeals/testing/components/ContactInformation.jsx

```diff
@@ -12,95 +12,97 @@ const ContactInformation = ({ formData }) => {
-              <p className="vads-u-margin-top--0p5">
-                <Link
-                  id="edit-mobile-phone"
-                  aria-label="Edit mobile phone number"
-                  to="/edit-mobile-phone"
-                >
-                  edit
-                </Link>
-              </p>
-              <h2 className="vads-u-width--auto">Email address</h2>
-              <span className="dd-privacy-hidden" data-dd-action-name="email">
-                {veteran.email}
-              </span>
-              <p className="vads-u-margin-top--0p5">
-                <Link
-                  id="edit-email"
-                  aria-label="Edit email address"
-                  to="/edit-email-address"
-                >
-                  edit
-                </Link>
-              </p>
-              <h2 className="vads-u-width--auto">Mailing address</h2>
-              <div className="dd-privacy-hidden" data-dd-action-name="street">
-                {addressLine1}
-              </div>
-              <div
-                className="dd-privacy-hidden"
-                data-dd-action-name="city, state and zip code"
+                edit
+              </Link>
+            </p>
+            <h2 className="vads-u-margin-top--4 vads-u-margin-bottom--0p5 vads-u-width--auto">
+              Email address
+            </h2>
+            <span className="dd-privacy-hidden" data-dd-action-name="email">
+              {veteran.email}
+            </span>
+            <p className="vads-u-margin-top--0p5">
+              <Link
+                id="edit-email"
+                aria-label="Edit email address"
+                to="/edit-email-address"
               >
-                {`${city}, ${stateCode} ${zipCode}`}
-              </div>
-              <p className="vads-u-margin-top--0p5">
-                <Link
-                  id="edit-address"
-                  aria-label="Edit mailing address"
-                  to="/edit-mailing-address"
-                >
-                  edit
-                </Link>
-              </p>
+                edit
+              </Link>
+            </p>
+            <h2 className="vads-u-margin-top--4 vads-u-margin-bottom--0p5 vads-u-width--auto">
+              Mailing address
+            </h2>
+            <div className="dd-privacy-hidden" data-dd-action-name="street">
+              {addressLine1}
             </div>
+            <div
+              className="dd-privacy-hidden"
+              data-dd-action-name="city, state and zip code"
+            >
+              {`${city}, ${stateCode} ${zipCode}`}
+            </div>
+            <p className="vads-u-margin-top--0p5 vads-u-padding-bottom--1">
+              <Link
+                id="edit-address"
+                aria-label="Edit mailing address"
+                to="/edit-mailing-address"
+              >
+                edit
+              </Link>
+            </p>
           </div>
-        </form>
+        </div>
```

Comment:
Not seeing the opening `<div>` for this closing `</div>`. `<form>` was deleted but did you mean to replace it with a `<div>`?

Reply:
The `<form>` was removed in line 15, so this might just look like it's being renamed. If you "hide whitespace" you'll see the changes as expected
<img width="250" alt="Screenshot 2023-09-08 at 6 51 42 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/136959/f7d07f75-dda0-4d24-aa8f-d96f3e0b72dd">